src/user_crate/mod.rs

```diff
@@ -12,6 +12,8 @@ pub(crate) use state_loaded::StateLoaded;
 pub(crate) use state_provisioned::StateProvisioned;
 
 use crate::PlRustError;
+#[cfg(feature = "verify")]
+use geiger;
```

Comment:
I am not fully satisfied with the geiger crate's implementation, but for basic user fn checks, it seemed sufficient as a start.